paper/basic_training.tex

```diff
@@ -744,6 +744,54 @@ \subsection{Integrators}
+
+Energy conservation is imperative in simulating the microcanonical (NVE) ensemble.
+This is a much trickier property to examine, and varies with different integrators.
+For instance, some classes of integrators better-preserve energy over short times, while others better-preserve energy at long times.
+The later is generally preferred, though it may necessitate other sacrifices such as greater energy fluctuations away from the desired, exact system energy.
+\todo[inline, color={green!20}]{JIM: Should show citations for this section where people analyze energy conservation of different integrators and make comment directing people to this.}
+
+Additionally, it is also desirable that an integrator be efficient, costing little computationally.
+Integrator cost mostly appears in the length of the timestep that may be taken while still avoiding discretization error. 
+As a general rule of thumb, the timestep must be at least as small as an order of magnitude less than the smallest timescale of motion present in the system (often this is bond vibration).
+However, depending on the accuracy of the integrator with respect to reproducing the true dynamics, a smaller timestep might be necessary.
+If the integrator requires a very small timestep to avoid discretization error, then the computational cost greatly increases.
+Hence, a truly ``good'' integrator allows for long timesteps while still achieving low discretization error.
+This has the added benefit of also reducing truncation error, which is proportional to the number of timesteps taken.
+
+The most commonly used integrators are variants of the Verlet algorithm. 
+Detailed discussion and derivation of such integrators may be found in Leach and Frenkel and Smit.
+\todo[inline, color={green!20}]{JIM: Insert citations and correct sections in these books}
+Such integrators are not applicable, however, for simulations involving stochastic dynamics.
+These simulations include application of a random force to each particle, and represent discretizations of either Langevin or Brownian dynamics.
+A detailed description of such stochastic dynamics may be found in McQuarrie, Chapter ??.
+\todo[inline, color={green!20}]{JIM: Need to cite the right chapter in McQuarrie, and others?}
+As detailed in section \ref{sec:thermostats}, it is common to apply temperature control through the use of Langevin dynamics.
+As a brief aside, this highlights the fact that integrators are tightly coupled to the chosen thermostat or barostat.
+Different combinations may demonstrate better performance and for expanded ensemble methods it is absolutely necessary to utilize an integrator specific to the selected temperature- or pressure-control algorithm.
+For simulating Langevin or other stochastic dynamics, the presence of random forces usually prevents the integrator from preserving phase-space volume.
```

Comment:
This sounds like it's saying that Langevin is bad in general, or at best, questionable. Probably need a sentence saying why it can work well in general for dynamics if you are not interested in kinetic/transport properties (e.g. we almost always use it for MD and as our thermostat).